### PR TITLE
Add exports-sql-data.sh

### PR DESCRIPTION
**Pull request by paragpoddar**

### CODE REVIEW

File: exports-sql-data.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Exports all databases data from a Cloud SQL instance to a Google Cloud Storage bucket as a SQL dump file
```
Comment (koladilip) [inline exports-sql-data.sh]: Add sample command to execute this script

**Review by koladilip — CHANGES_REQUESTED** [review]

File: exports-sql-data.sh (hunk 1 above)
Comment (paragpoddar) [reply exports-sql-data.sh]: OK

Comment (koladilip) [reply exports-sql-data.sh]: any update on this?

**Review by koladilip — CHANGES_REQUESTED** [review]

**Comment by paragpoddar** [comment]
will update.

**Comment by paragpoddar** [comment]
@koladilip  Updated.

**Comment by paragpoddar** [comment]
@koladilip Added documentation. Please check.

**Review by koladilip — APPROVED** [review]

**Pull request merged**